src/utils/collisions.ts

```diff
@@ -0,0 +1,44 @@
+import type { Collider, ColliderHandle, RigidBody, World } from '@dimforge/rapier3d-compat'
+import type { Group, Scene } from 'three'
+import type { Ref } from 'vue'
+
+interface CollisionSource {
+  collider: Collider
+  rigidBody: RigidBody | undefined
+};
+
+interface sourceTarget {
+  group: Group
```

Comment:
To match the returned value from `get3DGroupFromSource`, we should use an `Object3D` instead of `Collider`